src/BSON/Document.c

```diff
@@ -204,10 +234,8 @@ static PHP_METHOD(MongoDB_BSON_Document, get)
 
 	intern = Z_DOCUMENT_OBJ_P(getThis());
 
-	if (!php_phongo_document_get(intern, key, key_len, return_value, false)) {
-		// Exception already thrown
-		RETURN_NULL();
```

Comment:
This was redundant, as `null` is already the default value for the `return_value` zval.